### ISSUE TITLE
admin can delete an election

### ISSUE THREAD

**Issue by rhz** [benadida/helios-server#78]
it would make it possible to reuse election's short names for new elections (specially useful if there was a problem setting up the first one or it was just run as a test).

also, are these short names unique globally across users or only unique per user?

**Comment by benadida** [comment]
right now the short names are global, so you should use something that is somewhat specific to you.

**Comment by rhz** [comment]
I thought so... well, just consider it a feature request, users could make for a good namespace and deletion would be also quite nice.

**Comment by benadida** [comment]
yep, agreed, I think your suggestion would make things better indeed.

**Comment by benadida** [comment]
in progress in #161

**Comment by benadida** [comment]
choosing not to allow to delete elections
